common/web/gesture-recognizer/src/test/auto/browser/cases/canary.js

```diff
@@ -82,10 +82,14 @@ describe("'Canary' checks", function() {
       this.controller.recognizer.on('inputstart', fakeHandler)
       fireEvent();
 
+      // because of the mild optimization await when spinning up gesture-matching
```

Comment:
What is a "mild optimization await"? Can you clarify?

Reply:
Right, another thing that snuck in from the original changeset pre-spinoff.  Will fix.